pkgs/servers/mail/listmonk/frontend.nix

```
@@ -0,0 +1,29 @@
+{ yarn2nix-moretea
+, fetchFromGitHub
+}:
+
+yarn2nix-moretea.mkYarnPackage rec {
+  pname = "listmonk-frontend";
+
```

```suggestion

  src = fetchFromGitHub {
    owner = "knadh";
    repo = "listmonk";
    rev = "v${version}";
    sha256 = "....";
  };

```
please fill in the hash.